GSA-TTS/FAC-transition-site
# Updates Page Redesign

# Updates Page Redesign

Issue: https://github.com/GSA-TTS/FAC/issues/4204

## Changes:
Revamp the updates page to be many pages!

1. `/updates/` is now a roundup page with some cards and a paginated list of every update. 
2. `/updates/file-slug` is where each individual update gets a page.
3. Breadcrumbs have a framework, and can be added to specific pages rapidly from here on out.
4. Components galore, for use with other pages in the future.
5. A template/how-to is included for the new Update Page format.

## How to test:
For design/content, view the hosted branch [here](https://federalist-35af9df5-a894-4ae9-aa3d-f6d95427c7bc.sites.pages.cloud.gov/preview/gsa-tts/fac-transition-site/jp/updates-page-redesign//updates/).

For developers, try:
1. Switch to this branch and run everything normally
2.  Make sure the [updates page](http://localhost:8080/updates/) and some [individual updates](http://localhost:8080/updates/2024-08-21/) look right. 
3. Try adding a new update by following the new template/how-to, and verify that the page it creates looks correct.

## Screenshots:
The updates page:
![image](https://github.com/user-attachments/assets/ed80a3cb-634b-4950-825e-b7070870b1a8)

Updates page for mobile (slightly out of order, haven't found a good solution):
![image](https://github.com/user-attachments/assets/2c3327fe-5621-4b34-9cf9-9cc2d7a2f30d)

An individual update:
![image](https://github.com/user-attachments/assets/e46f9c6d-6ee1-4eb0-a88c-c1b328513351)

Individual update for mobile:
![image](https://github.com/user-attachments/assets/74699086-1761-47af-89de-3656e9b94d8f)

An Update with an image:
![image](https://github.com/user-attachments/assets/9ea5f602-dabc-4ad3-befa-502b099b2ca4)

The modal for an image:
![image](https://github.com/user-attachments/assets/62543169-fa23-4a1f-b6ff-b4309b92bd2b)
